source/_meta/_00-head.twig

```diff
@@ -5,21 +5,14 @@
 		<meta charset="UTF-8">
 		<meta name="viewport" content="width=device-width" />
 
+		<link rel="icon" type="image/png" href="{{ paths.assets }}/favicons/favicon-16x16.png" sizes="16x16">
+		<link rel="icon" type="image/png" href="{{ paths.assets }}/favicons/favicon-32x32.png" sizes="32x32">
```

Comment:
Per stuff like https://realfavicongenerator.net/, only two icons here seems light. Should we cover more stuff as a baseline? Nice icons on mobile devices seem like something we should have by default.

Reply:
Is there a reason we don't use `.ico` with multiple sizes included within? I suppose the HTML5 way is preferred here. 

This may be slightly outdated, but perhaps we should add some mobile/touch ones: https://css-tricks.com/favicon-quiz/

Reply:
Note: the webpack plugin has the ability to do ALL. This will be captured in FE docs eventually, but for now it's the two most common formats.

Reply:
Roger, but at this point it fills the webpack bundle output in terminal and is **really** distracting. I'll make a ticket to capture favicons approach.